content/ja/docs/concepts/workloads/autoscaling.md

```diff
@@ -0,0 +1,120 @@
+---
+tille: ワークロードの自動スケーリング
+description: >-
+  自動スケーリングによって、何らかのかたちでワークロードを自動的に更新できます。これによりクラスターはリソース要求の変化に対してより弾力的かつ効率的に対応できるようになります。
+content_type: concept
+weight: 40
+---
+
+<!-- overview -->
+
+Kubernetesでは、現在のリソース要求に応じてワークロードをスケールできます。
+これによりクラスターはリソース要求の変化に対してより弾力的かつ効率的に対応できるようになります。
+
+ワークロードをスケールするとき、ワークロードによって管理されるレプリカ数を増減したり、レプリカで使用可能なリソースをインプレースで調整できます。
+
+ひとつ目のアプローチは _水平スケーリング_ と呼ばれ、一方でふたつ目のアプローチは _垂直スケーリング_ と呼ばれます。
+
+ユースケースに応じて、ワークロードをスケールするには手動と自動の方法があります。
+
+<!-- body -->
+
+## ワークロードを手動でスケーリングする
+
+Kubernetesはワークロードの _手動スケーリング_ をサポートします。
+水平スケーリングは `kubectl` CLIを使用して行うことができます。
+垂直スケーリングの場合、ワークロードのリソース定義を `パッチ適用` する必要があります。
```

Comment:
```suggestion
垂直スケーリングの場合、ワークロードのリソース定義を _パッチ適用_ する必要があります。
```
In [original `en` content](https://github.com/kubernetes/website/blob/13cae23279b9a224b5b338384aa16ee92c1346c7/content/en/docs/concepts/workloads/autoscaling.md?plain=1#L28), the "_" syntax is used.